### ISSUE TITLE
not working any more

### ISSUE THREAD

**Issue by skarbovskiy** [phts/firefox-web-skype#7]
Starting from yesterday error alert
```
Oops, it looks like something went wrong. Click OK to automatically reload Skype for Web.
```
pops-up on loading when browsing from FF, works fine frome chrome.
Maybe they have added some new domains? :(

**Comment by lgblgblgb** [comment]
> Starting from yesterday error alert
> 
> ```
> Oops, it looks like something went wrong. Click OK to automatically reload Skype for Web.
> ```
> 
> pops-up on loading when browsing from FF, works fine frome chrome.
> Maybe they have added some new domains? :(

I have never used this extension, but simply the user-agent switcher which worked fine. Till now, the same error you get. So I think there is nothing wrong with this extension or any other, it was Microsoft who changed something, that the simple "just change the user-agent" trick does not work any more, and now it's not important which extension you use to achieve this goal. The other possibility: Firefox 83 itself broke the compatibility, I am not sure about this, but it was almost the same day I updated my Firefox (however, it can be checked by using an older Firefox to see, if it works for not, it that stopped working as well, it's a clear sign that it's nothing to do with Firefox changes either, and really Microsoft did something to prevent the "change the user agent" trick to work. Which is a shame on them ... :-/ But I can be wrong, and maybe really, indeed some new domains I used as well, and just those must be added ... Sorry about the long comment, I just wanted to add my two cents here, I'm not sure if it worked, though :) Oh, and sorry about my suboptimal English knowledge ;)

**UPDATE**: Using the console (ctrl-shift-k) network tab, I've added all the domains with wildcards to the user-agent switcher including ones like skypeassets.com and all the others, still it does not work ... So I have the suspect at least, that the reason is not a new domain, but I can't be sure ...

One interesting inspection that I can see my contacts, etc, even the number of new messages (and the head of the most up-to-date one, which is correct, checked on my phone with the skype app, and they're just few minutes old!) in the background "greyed out" thus it would work, but that stupid alert window appears and you can't get rid of it, only click on OK, which causes to reload the page. Now it's more and more looks like, it's Microsoft's way to prevent Firefox to be used, as it's clear it would work ...

**Comment by phts** [comment]
@skarbovskiy  @lgblgblgb  OMG, Skype seems to find the way how to end up with FF completely... I will investigate what's going on...

**Comment by phts** [comment]
@skarbovskiy @lgblgblgb Hmm just tried on FF83 Windows 7 - and it works for me. I can log in and open https://web.skype.com/. Please try again from scratch in private mode.

**Comment by skarbovskiy** [comment]
hm.. deleting all local storage data really resolves the issue for some random period of time (from 30 sec to 5 minutes as testing now). will try to produce more detailed report

**Comment by lgblgblgb** [comment]
Interesting idea! It seems it works in private mode (I haven't even thought about this, since I never used it in private browsing mode ...). At this point, I can't understand why it does make difference. I used web skype with simple "user agent switcher trick" since months, and **_never_** in private mode, but it seems now, since some days it only works in private mode? It sounds strange for me ...

**Comment by phts** [comment]
Hi @lgblgblgb 

> At this point, I can't understand why it does make difference.

This means that cache/session/local storage may affect Skype when determines supported browser. Of course private mode is not a solution, but workaround and shows that possible problem is in already stored data in normal mode.

Try to clean all cookies/cache/storage/etc related to skype and microsoft and log in again from scratch.

**Comment by StSav012** [comment]
> Try to clean all cookies/cache/storage/etc related to skype and microsoft and log in again from scratch.

Did not help me. The page gets stuck at sending request to `edge.skype.com`. And then the console reads that `app.js` is disabled by debugger. I don't see the messages in a new container (thanks to Firefox for the feature). Hope this helps.

**Comment by phts** [comment]
@StSav012 I cannot reproduce :disappointed:  For me it works.

I released new version 0.5.1 and added `edge.skype.com` for user-agent manipulation as well. But honestly speaking not sure if it helps :confused:

**Comment by chung1905** [comment]
Glad you updated it.

I have 2 devices, only one have this issue. I have to open skype in container to use it.

Thank you for good extension.
